cvmfs/loader.cc

```diff
@@ -801,11 +820,11 @@ int FuseMain(int argc, char *argv[]) {
 
   // Logging
   if (options_manager->GetValue("CVMFS_SYSLOG_LEVEL", &parameter))
-    SetLogSyslogLevel(String2Uint64(parameter));
+    SetLogSyslogLevel(atoi(parameter.c_str()));
```

Comment:
```suggestion
    SetLogSyslogLevel(String2Uint64(parameter));
```
change unrelated to this pr